tox.ini

```diff
@@ -0,0 +1,33 @@
+[tox]
+minversion = 3.14.6
+envlist =
+    py{38,39,310,311}-dj{32,41}-wagtail{41,50,51}
+    py{39,310,311}-dj{42}-wagtail{50,51}
```

Comment:
I think here we should take into account that dj32 doesn't support py3.11

Reply:
Ah OK, thanks for updating it. I wonder what's right now but I don't suppose it matters too much 😄